TDDPracticeTests/TDDPracticeTests/CalculateNumber/CalculatorTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FluentAssertions;
+using Xunit.Abstractions;
+
+namespace TestCasesProject.CalculatorCaseStudy
+{
+
+    public class CalculatorTests : IDisposable
+    {
+        private readonly ITestOutputHelper _output;
+        public CalculatorTests(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+
+        //dispose all running instances
+        public void Dispose()
+        {
+        }
+
+        [Theory]
+        [MemberData(nameof(TestData))]
+        public void Return_Sum_Of_Numbers(string numbers, int expectedResult)
+        {
+            //Act
+            int result = Calculator.Add(numbers);
+
+            //Assert
+            _output.WriteLine("Return sum of number is executed successfully.");
+            result.Should().NotBe(0).And.BePositive().And.Be(expectedResult, "because they have same values");
+        }
+
+        public static IEnumerable<object[]> TestData => new[]
```

Comment:
this is very outdated 
please look into how to do this in 2024

Reply:
Rewrite this code. Please check and let me know if changes needed.